.env.secrets

```diff
@@ -0,0 +1,59 @@
+# wallet
+WALLET_DATA=./wallet_data
+
+# global
+STAGE=dev
+APP_NAME=lifebank
+EOS_PRIV_KEY=${{ secrets.EOS_PRIV }}
```

Comment:
here

Reply:
@Edgar097  env files contain the values for environmental variables in local environments, for the development environment we will use github secrets instead of this file 

We don't do string interpolation of environmental variables here , such as `${{ secrets.EOS_PRIV }}` , this would be used in the github workflow yaml files, never in `.env` files